### PR TITLE
Update distribution.md

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

File: content/yaml-publishing/distribution.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ If your changes are sent to review automatically, but the field is still set to
 `Changes are sent for review automatically. The query parameter changesNotSentForReview must not be set.`
 {{</notebox>}}
 
-If you are getting a 400 error related to the app being in draft status, you need to promote your draft build to the next level up of testing tracks. Play Console will show you how to do this. You'll need to go through the steps, fill out questionnaires, upload various screenshots, and then after approval, you can move to the Alpha testing track, and Codemagic will successfully publish (publishing builds on Draft status is not supported).
+If you are getting a 400 error related to the app being in draft status, either publishing to draft can be enabled by **submit_as_draft: true** or there is a need to promote draft build to the next level up of testing tracks. Play Console will show you how to do this. You'll need to go through the steps, fill out questionnaires, upload various screenshots, and then after approval, you can move to the Alpha testing track, and Codemagic will successfully publish.
```
Comment (VeArnold) [inline content/yaml-publishing/distribution.md:101]: perhaps this wording would be better:

...either enable publishing to draft by setting the value of **submit_as_draft** to **true** or promote the draft build to the next level up of testing tracks...

I'd ask @helinanever to review the wording, other than that, LGTM

Comment (icarusdust) [reply content/yaml-publishing/distribution.md]: Changes made, Arnold!

Comment (helinanever) [reply content/yaml-publishing/distribution.md]: What @VeArnold suggested looks good to me in general but perhaps sth like this (if the changes do not change the intended meaning): 
...either enable publishing to draft by setting the value of submit_as_draft to true or promote the draft build up by a level to one of the testing tracks...

Comment (VeArnold) [reply content/yaml-publishing/distribution.md]: floats my boat

**Pull request merged**